db/seeds.rb

```diff
@@ -0,0 +1,11 @@
+messages = [
+  "Welcome to our app!",
+  "It is a pleasure having you in here.",
+  "Thank you for joining us.",
+  "Welcome to the community.",
+  "We hope you feel at home."
+]
+
+messages.each do |content|
+  Message.create(content: content)
+end
```

Comment:
- Nice job for seeding your table with greeting messages but unfortunately the migration file for creating the table doesn't exit at all